bootstrap/kubeadm/controllers/kubeadmconfig_controller_test.go

```diff
@@ -596,7 +596,7 @@ func TestReconcileIfJoinNodesAndControlPlaneIsReady(t *testing.T) {
 	}
 }
 
-func TestBootstrapTokenTTLExtension(t *testing.T) {
+func TestBootstrapTokenTTLExtension(t *testing.T) { //nolint
```

Comment:
Why is `nolint` needed here when it wasn't previously?

Reply:
Same as above